test/02-trigger-bisect-jobs.t

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
```

Comment:
It's unconventional to have a shell script as test for a python script. For now it looks simple enough but I guess soon we should go for pytest